### DISCUSSION TITLE
Erro em execução de projeto

### DISCUSSION THREAD

**Discussion by IzaqueL95** [pythonbrasil/pybr2022-site#129] [Q&A]
Olá pessoal, caso não seja o espaço adequado para esse tipo de discussão peço desculpas, mas gostaria de tirar uma duvida na execução do:  [pybr2022-site](https://github.com/pythonbrasil/pybr2022-site)  via coomando yarn develop

Estou obtendo esse erro: 
![image](https://user-images.githubusercontent.com/79032014/196016824-30d8b5bd-cfbc-43d2-b2e8-3c5a03c9bb2b.png)

Gostaria de contribuir com o projeto, segui o básico que estava no readMe, mas obtive o erro.

obrigado.

**Comment by felipecolen** [comment]
Fala @IzaqueL95 tudo bem?

Cara, imagino que seja algo relacionado ao ambiente windows, por exemplo, nesse @import tem as barras assim \ e /... 

Só pra validar, a versão do seu node é qual?

```
$ node -v
v16.17.1
```

Se puder tentar rodar esses comandos aqui e repassar...
```
npm install 
npm start
```

**Comment by IzaqueL95** [comment]
Opa @felipecolen tudo bem e contigo? agradeço seu retorno!

Em relação aos comandos iniciais que você solicitou:
![image](https://user-images.githubusercontent.com/79032014/196054788-5fa1a6ce-7e00-49f8-b185-73c33157d810.png)

Após o npm start:
![image](https://user-images.githubusercontent.com/79032014/196054826-8351f393-d35e-4d7d-b755-1bc81a3cc7f8.png)

O erro persiste, realmente não havia pensado na questão das barras, existe uma forma de corrigir?

**Reply by felipecolen** [reply]
Poderia tentar instalar esse cara aqui?

`npm install --global windows-build-tools`

**Reply by felipecolen** [reply]
Tem uma doc interessante aqui

https://www.gatsbyjs.com/docs/how-to/local-development/gatsby-on-windows/

**Reply by felipecolen** [reply]
conseguiu @IzaqueL95 ?

**Reply by IzaqueL95** [reply]
Não, o erro persiste.
